docs/quickstart.html

```diff
@@ -64,6 +72,28 @@ <h4 class="anchor-heading">
         <p>
             Once all services have successfully launched, you will have a basic Kafka environment running and ready to use.
         </p>
+
+        <h5>
+            Kafka with KRaft
+        </h5>
+
+        <p>
+            Generate a Cluster UUID
+        </p>
+
+        <pre class="line-numbers"><code class="language-bash">$ KAFKA_CLUSTER_ID = $(bin/kafka-storege.sh random-uuid)</code></pre>
```

Comment:
1. typo: kafka-stor[e]ge.sh -> kafka-storage.sh
2. in shell, variable assignment should has no space between "=", i.e.
`KAFKA_CLUSTER_ID=$(bin/kafka-storage.sh random-uuid)`

Reply:
Done.